src/Journal/JournalForm.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react'
+import { Link, useHistory } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import { v4 as uuidv4 } from 'uuid'
+import Button from '../common/Button'
+import HeadlineUnderline from '../common/HeadlineUnderline'
+
+export default function JournalForm({ createJournalEntry }) {
+  const currentDate = new Date()
+  const [title, setTitle] = useState('')
+  const [description, setDescription] = useState('')
+  const history = useHistory()
+
+  return (
+    <FormStyled onSubmit={handleSubmit}>
+      <HeadlineUnderline>
+        <h1>Eintrag erstellen</h1>
+      </HeadlineUnderline>
+      <DateHeadline>Datum</DateHeadline>
```

Comment:
I think Underline will be enough here, Normally there is just 1 underline I think, so headline is not needed to mention here

Reply:
I also have underlines for Text-Links, so for me it made sense to differentiate. I'll think about the naming again, though. I might be able to make the nameing more consistent and easier to understand.